3.4/graphs-pregel.md

```diff
@@ -154,9 +154,9 @@ The object returned by the `status` method might for example look something like
 ### Canceling an Execution / Discarding results
 
 To cancel an execution which is still running, and discard any intermediate results you can use the `cancel` method.
-This will immediately  free all memory taken up by the execution, and will make you lose all intermediary data. 
+This will immediately free all memory taken up by the execution, and will make you lose all intermediary data. 
 
-You might get inconsistent results if you cancel an execution while it is already in it's `done` state. The data is written
+You might get inconsistent results if you cancel an execution while it is already in its `done` state. The data is written
```

Comment:
Seems like there is no way to tell if it's really done (including writing the results)?

Reply:
I only know about `pregel.status(id)` but I don't know if that includes writing the results.

Reply:
@graetzer is it possible to determine when it's really done (computation + writing)?

Reply:
talked with Simon, he says it's possible to change it somehow so that you can differentiate between done processing and done writing (really done).